Bonsai/Program.cs

```diff
@@ -170,7 +172,11 @@ internal static int Main(string[] args)
             {
                 var bootstrapper = launchEditor ? (Bootstrapper)new EditorBootstrapper(editorRepositoryPath) : new ConsoleBootstrapper(editorRepositoryPath);
                 try { bootstrapper.RunAsync(Launcher.ProjectFramework, packageConfiguration, editorPath, editorPackageName).Wait(); }
-                catch (AggregateException) { return ErrorExitCode; }
+                catch (AggregateException ex)
+                {
+                    Console.Error.WriteLine(ex);
+                    return ErrorExitCode;
+                }
```

Comment:
I somehow feel like this was solved before, will dig through notes for 2.8.3.

Reply:
I believe you're thinking of https://github.com/bonsai-rx/bonsai/pull/1805, which was a slightly different scenario for a different exception path.

Reply:
Ah, that's right, and that reminds me why I think I left this here: if you run the bootstrapper from the command line, it will print the error twice, once from the console logger and again from the Console.Error here. I will try to verify this is the case.

Reply:
@PathogenDavid actually this is indeed indirectly related to #1805, which is a proposed fix to #1804 which discusses exactly this issue. #1805 was supposed to fix it since any package bootstrap errors will be routed to either a UI or the console logger standard error anyway.

Most likely what is happening here if you are hitting this are errors outside the NuGet client workflow, I would guess for example the assembly resolution and loading errors? Those would crash the process silently, which I agree is not great.

The best might be to revert #1805 by removing the `throw` statement after logging to make the behavior identical to the UI bootstrapper (i.e. if the exception is logged, it is not rethrown). In that case printing any other unhandled exceptions in the catch statement as you did at the top-level should not generate duplicate error traces.

Reply:
The above is also not entirely satisfactory since not throwing in the console bootstrapper will prevent returning a proper error exit code which can be useful for CLI evaluation. Maybe we just merge this anyway and reopen #1804 and I can submit a better PR to address this later.